.idea/.gitignore

```diff
@@ -0,0 +1,7 @@
+# Default ignored files
```

Comment:
could this just be `.gitignore` at root with:

```
.idea/
```

Reply:
sigh, I forgot that would be in here.

But at some point I want to figure out which is the actual right set of IDEA files to share, so I'll optimize this then.